argilla-frontend/translation/es.js

```diff
@@ -0,0 +1,272 @@
+    cancel: "Cancelar",
+    continue: "Continuar",
+    delete: "Eliminar",
+    tooltip: {
+      copyToClipboard: "Copiar en el portapapeles",
+      copyNameToClipboard:
+        "Copiar el nombre del conjunto de datos al portapapeles",
+      copyLinkToClipboard:
+        "Copiar enlace del conjunto de datos al portapapeles",
+      goToDatasetSettings: "Vaya a la configuración del conjunto de datos",
+      datasetSettings: "Configuración del conjunto de datos",
+    },
+  },
+  to_submit_complete_required: "Para enviar respuestas completas",
+  some_records_failed_to_annotate: "Algunos registros no lograron anotar",
+  changes_no_submit: "No envió sus cambios",
+  bulkAnnotation: {
+    recordsSelected:
+      "1 registro seleccionado | {count} registros seleccionados",
+    recordsViewSettings: "Tamaño de registro",
+    fixedHeight: "Records de colapso",
+    defaultHeight: "Expandir registros",
+    to_annotate_record_bulk_required: "No hay registro seleccionado",
+    select_to_annotate: "Seleccione todo",
+    pageSize: "Tamaño de página",
+    selectAllResults: "Seleccione todos los registros coincidentes {Total}",
+    haveSelectedRecords: "Has seleccionado todos los registros {Total}",
+    actionConfirmation: "Confirmación de acción a granel",
+    actionConfirmationText:
+      "Esta acción afectará {total} registros, ¿Quiere continuar?",
+    allRecordsAnnotated: "Los registros {Total} han sido {Action}",
+    affectedAll: {
+      submitted: "enviado",
+      discarded: "descartado",
+      draft: "borrador",
+    },
+  },
+  shortcuts: {
+    label: "Atajos",
+    pagination: {
+      go_to_previous_record: "Anterior (←)",
+      go_to_next_record: "Siguiente (→)",
+    },
+  },
+  questions_form: {
+    validate: "Validar",
+    clear: "Claro",
+    reset: "Reiniciar",
+    discard: "Descartar",
+    submit: "Entregar",
+    draft: "Guardar como borrador",
+    write: "Escribir",
+  },
+  sorting: {
+    addOtherField: "+ Agregar otro campo",
+    suggestion: {
+      score: "Puntuación de sugerencias",
+      value: "Valor de sugerencia",
+    },
+    response: "Valor de respuesta",
+    record: "general",
+    metadata: "Metadatos",
+  },
+  suggestion: {
+    agent: "\nagente: {agent}",
+    score: "\npuntaje: {score}",
+    tooltip: "Esta pregunta contiene una sugerencia {agente} {stork}",
+    filter: {
+      value: "Valores de sugerencia",
+      score: "Puntaje",
+      agent: "Agente",
+    },
+    plural: "Sugerencias",
+    name: "Sugerencia",
+  },
+  similarity: {
+    "record-number": "Número de registro",
+    findSimilar: "Buscar similares",
+    similarTo: "Similar a",
+    similarityScore: "Puntuación de similitud",
+    similarUsing: "Similar usando",
+    expand: "Expandir",
+    collapse: "Colapsar",
+  },
+  spanAnnotation: {
+    shortcutHelper: "Presiona 'Shift' para seleccionar solo caracteres",
+    notSupported: "La anotación en selección no es compatible con su navegador",
```

Comment:
```suggestion
    notSupported: "La anotación de tipo span no es compatible con su navegador",
```

Reply:
Span questions are translated above as "Selección", so probably here we want something like "La anotación de tipo selección no es..." to keep it consistent.